itou/templates/users/details.html

```diff
@@ -153,15 +141,8 @@ <h2>Données administratives</h2>
                                         {% if profile.pole_emploi_id or profile.lack_of_pole_emploi_id_reason %}
                                             {% if profile.pole_emploi_id %}
                                                 <strong>{{ profile.pole_emploi_id }}</strong>
-                                                <button class="btn-link"
-                                                        data-bs-toggle="tooltip"
-                                                        data-bs-placement="top"
-                                                        data-bs-trigger="manual"
-                                                        data-bs-title="Copié!"
-                                                        data-it-clipboard-button="copy"
-                                                        data-it-copy-to-clipboard="{{ profile.pole_emploi_id }}">
-                                                    <i class="ri-file-copy-line"></i>
-                                                </button>
+                                                {% matomo_event "gps" "clic" "copied_user_pe_id" as matomo_event_attrs %}
+                                                {% include 'includes/copy_to_clipboard.html' with content=profile.pole_emploi_id css_classes="btn-link" matomo_event_attrs=matomo_event_attrs text=" " %}
```

Comment:
Je me demande si on ne devrait pas modifier l'include pour ne pas mettre de span quand text est vide (cela nécessiterait d'enlever le `|default:"Copier"` mais il n'est pas utilisé si souvent) ou alors rajouter un `{% if not only_icon|default:False %}` autour de la balise span ?

Reply:
Bonne idée !
Je viens de changer.

Reply:
Finalement je préfère ton idée de only_icon. Je change.